src/octopeer-github/main/main.ts

```diff
@@ -1,15 +1,5 @@
-/// <reference path="DatabaseAdaptable/DatabaseAdaptable.ts"/>
-/// <reference path="DatabaseAdaptable/DatabaseConsoleLogOnly.ts"/>
-/// <reference path="ElementSelectionBehaviour/ElementSelectionBehaviour.ts"/>
-/// <reference path="ElementSelectionBehaviour/ButtonsElementSelectionBehaviour.ts"/>
-/// <reference path="ElementEventBinding/ElementEventBinding.ts"/>
-/// <reference path="ElementEventBinding/ClickElementEventBinding.ts"/>
-
-let databaseAdapterInterfaceImplementer: DatabaseAdaptable;
-let buttonSelector: ElementSelectionBehaviour;
-let buttonClickedLogging: ElementEventBinding;
-
-databaseAdapterInterfaceImplementer = new DatabaseConsoleLogOnly();
-buttonSelector = new ButtonsElementSelectionBehaviour(databaseAdapterInterfaceImplementer);
-buttonClickedLogging = new ClickElementEventBinding(buttonSelector);
-
+$(document).ready(function() {
+    let controller = new Controller();
+    console.log(controller); // suppress unused
```

Comment:
Then TSLint gives a 'no-unused-expression' error because it's not stored into a variable XD